YoutubeExplodeTests/YoutubeClientPlaylistParsingTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using YoutubeExplode.Services;
+
+namespace YoutubeExplode.Tests
+{
+    [TestClass]
+    public class YoutubeClientPlaylistParsingTests
```

Comment:
I used a mock here because I wanted to avoid failing tests because the actual playlist was changed/deleted.
You are right about youtube changing things though.
I would leave this test but add an additional one to the intergration tests. I just have to find a playlist that does not change too often.
What do you think?